html/index.php

```diff
@@ -129,7 +129,7 @@
   <link href="css/leaflet.awesome-markers.css" rel="stylesheet" type="text/css" />
   <link href="css/select2.min.css" rel="stylesheet" type="text/css" />
   <link href="<?php echo($config['stylesheet']);  ?>?ver=291727421" rel="stylesheet" type="text/css" />
-  <link href="css/<?php echo $config['site_style']; ?>.css?ver=632417642" rel="stylesheet" type="text/css" />
+  <link href="css/<?php echo $config['site_style']; ?>.css?ver=632417692" rel="stylesheet" type="text/css" />
```

Comment:
This isn't needed when no css files are changed or did you miss committing that change?

Reply:
:) Did then reverted them back... so forgot to revert this back too :)